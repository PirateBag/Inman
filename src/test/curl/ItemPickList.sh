#!/bin/bash
curl -d '{ "idToSearchFor" : "1" }' -H "Content-Type: application/json" -X POST http://localhost:8080/itemPick/all