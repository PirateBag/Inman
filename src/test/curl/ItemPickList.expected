{
  "responseType" : "ADD",
  "errors" : [ ],
  "data" : [ {
    "id" : 1,
    "activityState" : "NONE",
    "external" : "W-001, 36 in red wagon"
  }, {
    "id" : 2,
    "activityState" : "NONE",
    "external" : "W-002, red painted wagon body"
  }, {
    "id" : 3,
    "activityState" : "NONE",
    "external" : "W-003, Front Wheel assembly"
  }, {
    "id" : 4,
    "activityState" : "NONE",
    "external" : "W-004, 24 in handle"
  }, {
    "id" : 5,
    "activityState" : "NONE",
    "external" : "W-005, 8 in wheel"
  }, {
    "id" : 6,
    "activityState" : "NONE",
    "external" : "W-006, Rear Wheel assembly"
  }, {
    "id" : 7,
    "activityState" : "NONE",
    "external" : "W-007, Rear wheel bracket"
  } ]
}